frontend/src/CMSContainer/CMSContainer.js

```diff
@@ -1,13 +1,31 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import ContentArea from '../ContentArea';
 import Sidebar from '../Sidebar';
 import { Wrapper } from './styled';
+import Loading from './Loading';
 
-const CMSContainer = () => (
-  <Wrapper>
-    <Sidebar />
-    <ContentArea />
-  </Wrapper>
-);
+const CMSContainer = props => {
+  const path = props.match.params.path || '';
+  let content = <Loading />;
+  if (props.cmsReady) {
+    content = <ContentArea path={path} />;
+  }
+  return (
+    <Wrapper>
+      <Sidebar />
+      {content}
+    </Wrapper>
+  );
+};
 
```

Comment:
React way would be using ternary operator:
```js
{props.cmsReady ? <ContentArea path={path} /> : <Loading />}
```
Then you can get rid of `let content` and `if`